proposal/comp110_architecture_proposal.tex

```diff
@@ -17,58 +17,58 @@ \section*{Topic}
 
 \section*{Paper 1}
 % This is an example! Replace the details with a paper relevant to your chosen topic.
 \begin{description}
-\item[Title:] A Mathematical Theory of Communication
-\item[Citation:] \cite{shannon}
-\item[Abstract:] ``The recent development of various methods of modulation such as PCM and PPM which exchange bandwidth for signal-to-noise ratio has intensified the interest in a general theory of communication. A basis for such a theory is contained in the important papers of Nyquist and Hartley on this subject. In the present paper we will extend the theory to include a number of new factors, in particular the effect of noise in the channel, and the savings possible due to the statistical structure of the original message and due to the nature of the final destination of the information.''
-\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6773024}
-\item[Full text link:] \url{http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=6773024}
-\item[Comments:] This is a seminal article, which laid the foundations for what we now call information theory.
-	It has been cited more than $76\,000$ times, making it one of the most-cited articles in computer science.
+\item[Title:] Rhythm-based level generation for 2D platformers
+\item[Citation:] \cite{Smith:2009}
+\item[Abstract:] ``We present a rhythm-based method for the automatic generation of levels for 2D platformers, where the rhythm is that which the player feels with his hands while playing. Levels are created using a grammar-based method: first generating rhythms, then generating geometry based on those rhythms. Generation is constrained by a set of style parameters tweakable by a human designer. The approach also minimizes the amount of content that must be manually authored, instead relying on geometry components that are included in the level designer's tileset and a set of jump types. Our results show that this method produces an impressive variety of levels, all of which are fully playable.''
+\item[Web link:] \url{http://dl.acm.org/citation.cfm?id=1536548}
+\item[Full text link:] \url{https://users.soe.ucsc.edu/~ejw/papers/smith-platformer-generation-fdg2009.pdf}
+\item[Comments:] This is a conference article that discusses the possibilities of creating playable levels with the incorporation of rhythm generation for geometry placed platforms.
 \end{description}
 
 \section*{Paper 2}
 \begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
+\item[Title:] An approach to level design using procedural content generation and difficulty curves
```

Comment:
An interesting paper, however it seems to focus on 3D platformers. The methods may apply to 2D games as well, but this paper is possibly less relevant than the others.